src/components/JoinUsBar/JoinUsBar.css

```diff
@@ -0,0 +1,34 @@
+
+/*Problema da risolvere: centrare e ordine il bottone e il testo*/
+.join_us_bar {
+    background-color: #fc3f1a; /*#120068;*/
+    height: 156px;
+    justify-content:center;
+    align-items: center;
+    text-align: center;
+}
+
+.JSB_button {
+  background-color: white; /*#fc3f1a;*/
+  color: black;
+  padding: 0.4em 5em;
+  border-radius: 1em;
```

Comment:
Qui usa "border-radius: 50%"

Reply:
Impostando "border-radius" in percentuale o in pixel rendo ovale il bottone o eccessivamente squadrato. 
(Ovviamente, ho provato più valori)